packages/block-library/src/table/edit.js

```diff
@@ -407,13 +458,45 @@ export default class TableEdit extends Component {
 							onChange={ this.onChangeFixedLayout }
 						/>
 					</PanelBody>
+					<PanelColorSettings
+						title={ __( 'Color Settings' ) }
+						initialOpen={ false }
+						colorSettings={ [
+							{
+								value: backgroundColor.color,
+								onChange: setBackgroundColor,
+								label: __( 'Background Color' ),
+							},
+							{
+								value: textColor.color,
+								onChange: setTextColor,
+								label: __( 'Text Color' ),
+							},
+						] }
+					>
+						<ContrastChecker
```

Comment:
Contrast checker doesn't work so well with the stripes variation, since on some rows the contrast might be fine, but on others the contract could be problematic. One way to solve could be to pass through multiple colors in an array? Maybe something like an `additionalColors` prop. Alternatively, use two contrast checkers? That could get confusing.